src/items.rs

```diff
@@ -527,7 +527,7 @@ impl<'a> FmtVisitor<'a> {
         };
         let rewrite = format_struct(&self.get_context(), struct_parts, self.block_indent, None)
             .map(|s| if is_tuple { s + ";" } else { s });
-        self.push_rewrite(struct_parts.span, rewrite);
+        self.push_rewrite(struct_parts.span, rewrite.ok());
```

Comment:
After majority of `visit_***` functions are refactored, we'll have to report RewriteError instead of calling `rewrite.ok()`